install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e # everything must succeed.
+echo "[-] install.sh"
+
+python=/usr/bin/python3.6
```

Comment:
Does this assume python 3.6 is installed?

Reply:
yup. I've opened another PR with a test script and travis integration but your test suite has a few failing cases. Travis will test against 3.4, 3.5 and 3.6, which roughly correspond to the LTS Ubuntu releases since 12.04 to 18.04 - I think that's reasonable.

To make the python version here more flexible we can just do a `$(which python3)` and then issue a warning if the `x` in `python --version` (3.x.y) is not supported.

I didn't know if you target a specific Python3 or not

Reply:
3.5+ is required, post merge I can look at integrating this install script with the Django installation script we have so that it can do the full install. We would also need to update the base requirements to include venv and dropp virtualenv/virtualenvwrapper.

Reply:
yes, I have an unpushed change around here for that ...